### PR TITLE
Improved README Quick Start Steps

### PR DESCRIPTION
**Pull request by king-jam**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,15 +2,31 @@
...
+  $ ./build-package.bash quanta-t41 master
+  ```
+  
 * Register a new SKU with the package
-```
-$ curl -X POST -F file=@tarballs/sku_pack_directory_subname.tar.gz localhost:8080/api/common/skus/pack
-```
+
+  ```
+  $ curl -X POST -F file=@tarballs/sku_pack_directory_subname.tar.gz localhost:8080/api/common/skus/pack
```
Comment (zyoung51) [inline README.md:29]: @brianparry I just noticed that this is still referencing /common.  Were we changing references to explicitly say 2.0?

Comment (king-jam) [reply README.md:29]: Left that how I found it but would agree that it should probably be updated

**Review by zyoung51 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (brianparry) [reply README.md:29]: @zyoung51 @king-jam Yeah - we need to change /api/common/ to /api/current/.  We probably need to go through all of on-skupacks and update api references to /api/current

**Review by anhou — APPROVED** [review]
:+1:

**Pull request merged**